src/components/settings-menu/NotificationLoadingIndicator.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import Spinner from '@/components/Spinner';
+import styled from '@/styled-thing';
+import { IS_IOS } from '@/env';
+import { useTheme } from '@/theme';
+import { ActivityIndicator } from 'react-native';
+
+const ActivityIndicatorIos = styled(ActivityIndicator).attrs({
```

Comment:
I'm using SPinner on Android and native Activity indicator on iOS

Reply:
not rel: but we should totally standardize this at some point

Reply:
Agreed!